packages/babel-preset-kyt-core/README.md

```diff
@@ -35,6 +35,21 @@ $ yarn add @babel/runtime
 $ npm i --save-exact @babel/runtime
 ```
 
+- `typescript (Boolean)` option to `true` to automatically load [`@babel/preset-typescript`](https://babeljs.io/docs/en/babel-preset-typescript).
```

Comment:
Because `@babel/preset-typescript` is an `optionalDependency`, should there be a quick note about also manually installing in user dependencies?

Reply:
Thanks @delambo — added in de1258b.